<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meown - Alimentador inteligente para pets</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="imgHome">
      <property name="geometry">
       <rect>
        <x>-14</x>
        <y>-10</y>
        <width>421</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/src/img/home_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAction">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 127);
font: 9pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ativar alimentação manual</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Meown - Alimentador de Pets</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Conexão</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>361</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="status_conexao">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>17</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 25 17pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 127);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffff7f;&quot;&gt;Status: Desconectado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 25 15pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Porta:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="box_serial"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 25 15pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Velocidade:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="box_velocidade"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlug">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);
font: 25 15pt &quot;Bahnschrift Light&quot;;</string>
         </property>
         <property name="text">
          <string>Conectar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="imgConexao">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-8</y>
        <width>411</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/src/img/connexao_image.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>imgConexao</zorder>
     <zorder>layoutWidget</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Estatisticas</string>
     </attribute>
     <widget class="QLabel" name="imgEstatisticas">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>411</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/src/img/estatisticas_image.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabelaStats">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Tempo de refeição</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Vezes que se alimentou</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Nível de comida</string>
       </property>
      </row>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="Sobre">
     <attribute name="title">
      <string>Sobre</string>
     </attribute>
     <widget class="QLabel" name="imgSobre">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/src/img/sobre_image.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="imgEdu">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/src/img/edu.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="imgNeto">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/src/img/neto.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEmailNeto">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 0, 127);
font: 9pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEmailEdu">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
font: 9pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSite">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);
font: 9pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Site</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGithub">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 255);
font: 9pt &quot;Bahnschrift Light&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Github</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
